src/browser/ui/__tests__/ReactDOMComponent-test.js

```diff
@@ -775,4 +775,39 @@ describe('ReactDOMComponent', function() {
     });
   });
 
+  describe('option update', function(){
+    var container = document.createElement('div');
```

Comment:
These should be in a beforeEach so that the container is reinitialized before each spec.